config/core.extension.yml

```diff
@@ -3,6 +3,7 @@ module:
   big_pipe: 0
   block: 0
   breakpoint: 0
+  components: 0
```

Comment:
@rupl This needs a config import to enable the Components module
`drush cex`
https://drushreference.com/drush-8x/config/config-export/

Reply:
wouldn't I be doing a `drush cim` to import the config in this PR?

Reply:
Oops, yes, import